### PR TITLE
Changes to allow codebuild to create and run dockerfiles

### PR DESCRIPTION
**Pull request by kendase3**
Addresses OVER-7527 and OVER-7531

### CODE REVIEW

**Comment by kendase3** [comment]
They just swapped the UI and apparently I can't currently save changes to my gondar-windows target to use my in-aws-built dockerfile.  But these changes worked for the linux one so should be good.  I'm pretty sure with this setup we could have the dockerfiles re-run on every PR if that's what we want.

File: codebuild/buildlinux.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - echo Logging in
+      - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
```
Comment (nicholasbishop) [inline codebuild/buildlinux.yml]: Why is this command wrapped in `$(...)`?

Comment (kendase3) [reply codebuild/buildlinux.yml]: the aws example did that; maybe to resolve the env vars?

Comment (nicholasbishop) [reply codebuild/buildlinux.yml]: Seems strange but I'll live.

Comment (kendase3) [reply codebuild/buildlinux.yml]: i'll try without and see what happens

Comment (kendase3) [reply codebuild/buildlinux.yml]: nevermind u love it like this

**Pull request merged**